### PR TITLE
consensus/docs: draft UTxO HD design document for review

### PR DESCRIPTION
**Pull request by nfrisby**

### CODE REVIEW

**Comment by nfrisby** [comment]
Status update: I pushed up commits addressing the latest comments I received on Slack. So this document won't be changing (except maybe me addressing some TODOs) until I receive feedback on this PR.

File: ouroboros-consensus/docs/utxo-hd.md [hunk 1]
```diff
@@ -0,0 +1,719 @@
...
+
+## Other large parts of the ledger state
+
+The UTxO set is just the first part of the Shelley ledger that we will relocate
+to disk. It's likely to always be the largest part, so this is the most
+important and urgent. Soon enough, though, other parts will also grow too large
+for in-memory data structures.
+
+***Design Decision*** All parts of the Shelley larger proportional to the UTxO
+set and/or to the number of stake pools will be stored exclusively on-disk.
```
Comment (JaredCorduan) [inline ouroboros-consensus/docs/utxo-hd.md:514]:
```suggestion
set and/or to the number of registered stake credentials will be stored exclusively on-disk.
```

**Review by Jasagredo — COMMENTED** [review]
I think we do not want to merge this document anymore. However I think I can extract some info to enrich the UTxO-HD document.

**Comment by nfrisby** [comment]
Closing as stale.

**Pull request closed without merging**